docs/blog/2018-03-07-why-we-created-the-plugin-library/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Why we created the Gatsby plugin library
+date: 03-07-2018
+author: Shannon Soper
+---
+
+## Why build a plugin / package library
+
+In over 60 interviews with Gatsby users, plugin discoverability appeared as a top pain point (see [blog post on the Gatsby UX research program](/2017-12-20-introducing-the-gatsby-ux-research-program/) for a full list of pain points). So we built a plugin / package library [here](/packages/)!
+
+In this article, I want to explain how UX research drove the development and design of this plugin library and invite you all to follow and contribute as the library continues to improve.
+
+
+## Jobs to be done
+
+After talking to over 60 Gatsby users, I compiled what’s called an “empathy map” in UX design to put myself in the shoes of Gatsby users; basically, this map describes a summary of what the typical Gatsby user experiences when searching for plugins.
+
+![Empathy map](plugin-map.png)
+
+Based on the empathy map above, we identified five “jobs” that Gatsby users want done regarding plugins and packages:
+* Make it easy to search for and find the best, most recent, and relevant plugin for their site.
+    * Scrollable, searchable list of plugins with a search bar at the top, tags attached to each plugin, and accurate information on version compatibility, popularity, maintenance, stability.
```

Comment:
remove as per above comment. perhaps as per below comment, relevant features could be listed under the "learning" if they were inspired by other libraries

Reply:
Cool. I have a really detailed list of the strengths and weaknesses of the other sites--helpful to include? Too much? Or I can just add the features that we were inspired by...

Reply:
I'd say definitely include this. Plugin search is a pretty common pattern. There's a number of other communities that have one or might want one and you could contribute a lot here by recounting your research and analysis.

Reply:
done